ament_cmake_core/cmake/core/package_xml_2_cmake.py

```diff
@@ -111,7 +111,9 @@ def generate_cmake_code(package):
     lines.append('set(_AMENT_PACKAGE_NAME "%s")' % package.name)
     for (k, v) in variables:
         lines.append('set(%s_%s %s)' % (package.name, k, v))
-    return lines
+    # Ensure backslashes are replaced with forward slashes because CMake cannot
+    # parse files with backslashes in it.
+    return [l.replace('\\', '/') for l in lines]
```

Comment:
I am pretty sure that CMake can handle backslashes but they might need to be escaped. Since `\\\\` might become pretty ugly we should keep the forward slash.